sencha-workspace/SlateAdmin/app/controller/People.js

```diff
@@ -942,5 +947,26 @@ Ext.define('SlateAdmin.controller.People', {
             personRecord = person;
             _finishSelectPerson();
         }
+    },
+
+    createPerson: function() {
+        var me = this,
+            manager = me.getManager(),
+            grid, store, person;
+
+        Ext.suspendLayouts();
+        me.getNavPanel().expand();
+        me.application.getController('Viewport').loadCard(manager);
+        Ext.resumeLayouts(true);
+
+        grid = me.getGrid();
+        store = grid.getStore();
+        person = store.add({
```

Comment:
don't create variables for things used once, also always use the controller's reference to the store rather than going through the UI when available, as it is available earlier in the application lifecycle and not dependent on UI state: `me.getPeoplePeopleStore().add({});`